src/AutomationTests/TimedExecutionWrapper.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace Axe.Windows.AutomationTests
+{
+    /// <summary>
+    /// A simple wrapper to bring our timeout behavior under our own control. Given a max allowed time
+    /// and an Action, it spins up a thread, runs the Action on that thread, and reports the outcome
+    /// via the Completed and CaughtException properties. The same object can be used to run multiple
+    /// Actions (or the same Action multiple times), as long as they all share the same timeout.
+    /// </summary>
+    internal class TimedExecutionWrapper
+    {
+        private static readonly TimeSpan SleepTime = TimeSpan.FromMilliseconds(250);
```

Comment:
Guessing this was chosen somewhat arbitrarily?

Reply:
Yes. It's not special